python/pyiceberg/catalog/hive.py

```diff
@@ -233,24 +299,47 @@ def create_table(
             ValueError: If the identifier is invalid
         """
         database_name, table_name = self.identifier_to_database_and_table(identifier)
-        current_time_millis = int(time.time())
+        seconds_since_epoch = int(time.time() * 1000)
+
+        location = self._resolve_table_location(location, database_name, table_name)
+
+        metadata_location = f"{location}/metadata/{uuid.uuid4()}.metadata.json"
```

Comment:
In Java, we add a version to the start of the filename so that we can tell the order. Can you do that here as well?

Reply:
Great suggestion, it is also in the spec. Added 👍🏻